src/main/java/org/opentripplanner/gtfs/graphbuilder/GtfsModule.java

```diff
@@ -355,6 +355,9 @@ private GtfsMutableRelationalDao loadBundle(GtfsBundle gtfsBundle) throws IOExce
       transferRule.getFromLegGroupId().setAgencyId(reader.getDefaultAgencyId());
       transferRule.getToLegGroupId().setAgencyId(reader.getDefaultAgencyId());
     }
+    for (var transferRule : store.getAllEntitiesForType(FareLegRule.class)) {
+      transferRule.getFareProductId().setAgencyId(reader.getDefaultAgencyId());
+    }
```

Comment:
What is this change?

Reply:
Setting the correct agency id, which we convert to a feed id, was wrong before.